2024-08-29 23:27:24,680 - __main__ - INFO - Starting KNN Evaluation
2024-08-29 23:27:24,680 - __main__ - ERROR - Error evaluating KNN model:name 'paramet_tune' is not defined
2024-08-29 23:27:24,680 - __main__ - INFO - Starting Random Forest Classifier Evaluation
2024-08-29 23:27:24,680 - __main__ - ERROR - Error evaluating Random Forest Classifier model:RandomForest_tuner.__init__() takes 3 positional arguments but 5 were given
2024-08-29 23:27:24,680 - __main__ - INFO - Starting XGBoost Classifier Evaluation
2024-08-29 23:27:24,680 - __main__ - ERROR - Error evaluating XGBoost Classifier model:name 'paramet_tune' is not defined
2024-08-29 23:27:24,680 - __main__ - INFO - Starting SVM Classifier Evaluation
2024-08-29 23:27:24,681 - __main__ - ERROR - Error evaluating SVM model:The 'scoring' parameter of cross_validate must be a str among {'completeness_score', 'recall_micro', 'f1_macro', 'roc_auc_ovr_weighted', 'roc_auc_ovr', 'explained_variance', 'neg_mean_poisson_deviance', 'neg_mean_squared_error', 'neg_root_mean_squared_error', 'jaccard_weighted', 'neg_root_mean_squared_log_error', 'precision', 'positive_likelihood_ratio', 'jaccard', 'neg_median_absolute_error', 'homogeneity_score', 'neg_mean_absolute_error', 'matthews_corrcoef', 'recall_samples', 'f1', 'mutual_info_score', 'neg_mean_absolute_percentage_error', 'accuracy', 'recall_weighted', 'roc_auc_ovo', 'top_k_accuracy', 'normalized_mutual_info_score', 'precision_macro', 'f1_samples', 'jaccard_samples', 'neg_mean_squared_log_error', 'r2', 'rand_score', 'recall', 'fowlkes_mallows_score', 'max_error', 'balanced_accuracy', 'neg_negative_likelihood_ratio', 'f1_micro', 'precision_micro', 'precision_weighted', 'jaccard_macro', 'neg_log_loss', 'recall_macro', 'neg_mean_gamma_deviance', 'roc_auc', 'f1_weighted', 'adjusted_rand_score', 'jaccard_micro', 'precision_samples', 'adjusted_mutual_info_score', 'roc_auc_ovo_weighted', 'v_measure_score', 'average_precision', 'neg_brier_score'}, a callable, an instance of 'list', an instance of 'tuple', an instance of 'dict' or None. Got array([0, 0, 0, ..., 0, 0, 0]) instead.
2024-08-29 23:27:24,681 - __main__ - INFO - Starting Naive Bayes Classifier Evaluation
2024-08-29 23:27:24,681 - __main__ - ERROR - Error evaluating Naive Bayes model:The 'scoring' parameter of cross_validate must be a str among {'completeness_score', 'recall_micro', 'f1_macro', 'roc_auc_ovr_weighted', 'roc_auc_ovr', 'explained_variance', 'neg_mean_poisson_deviance', 'neg_mean_squared_error', 'neg_root_mean_squared_error', 'jaccard_weighted', 'neg_root_mean_squared_log_error', 'precision', 'positive_likelihood_ratio', 'jaccard', 'neg_median_absolute_error', 'homogeneity_score', 'neg_mean_absolute_error', 'matthews_corrcoef', 'recall_samples', 'f1', 'mutual_info_score', 'neg_mean_absolute_percentage_error', 'accuracy', 'recall_weighted', 'roc_auc_ovo', 'top_k_accuracy', 'normalized_mutual_info_score', 'precision_macro', 'f1_samples', 'jaccard_samples', 'neg_mean_squared_log_error', 'r2', 'rand_score', 'recall', 'fowlkes_mallows_score', 'max_error', 'balanced_accuracy', 'neg_negative_likelihood_ratio', 'f1_micro', 'precision_micro', 'precision_weighted', 'jaccard_macro', 'neg_log_loss', 'recall_macro', 'neg_mean_gamma_deviance', 'roc_auc', 'f1_weighted', 'adjusted_rand_score', 'jaccard_micro', 'precision_samples', 'adjusted_mutual_info_score', 'roc_auc_ovo_weighted', 'v_measure_score', 'average_precision', 'neg_brier_score'}, a callable, an instance of 'list', an instance of 'tuple', an instance of 'dict' or None. Got array([0, 0, 0, ..., 0, 0, 0]) instead.
2024-08-29 23:27:24,681 - __main__ - INFO - Starting CatBoost Classifier Evaluation
2024-08-29 23:27:24,681 - __main__ - ERROR - Error evaluating CatBoost model:Catboost_tuner.__init__() takes 3 positional arguments but 5 were given
2024-08-29 23:27:24,681 - __main__ - INFO - Starting LOF Evaluation
2024-08-29 23:27:24,789 - __main__ - ERROR - Error evaluating LOF model:"None of [Index([     1,      3,      4,      5,      8,      9,     10,     11,     13,\n           14,\n       ...\n       594630, 594631, 594633, 594635, 594636, 594637, 594638, 594639, 594640,\n       594641],\n      dtype='int64', length=475714)] are in the [columns]"
2024-08-29 23:27:24,790 - __main__ - INFO - Starting PCA Evaluation
2024-08-29 23:27:25,488 - __main__ - ERROR - Error evaluating PCA model:too many values to unpack (expected 3)
2024-08-29 23:27:25,488 - __main__ - INFO - Starting Isolation Forest Evaluation
2024-08-29 23:27:25,488 - __main__ - ERROR - Error evaluating Isolation Forest model:name 'paramet_tune' is not defined
2024-08-29 23:27:25,488 - __main__ - INFO - Starting DBSCAN Evaluation
2024-08-29 23:27:25,546 - __main__ - ERROR - Error evaluating DBSCAN model:"None of [Index([     1,      3,      4,      5,      8,      9,     10,     11,     13,\n           14,\n       ...\n       594630, 594631, 594633, 594635, 594636, 594637, 594638, 594639, 594640,\n       594641],\n      dtype='int64', length=475714)] are in the [columns]"
2024-08-29 23:27:25,547 - __main__ - INFO - Starting COPOD Evaluation
2024-08-29 23:27:30,142 - __main__ - ERROR - Error evaluating COPOD model:too many values to unpack (expected 3)
2024-08-29 23:27:30,143 - __main__ - INFO - Starting ECOD Evaluation
2024-08-29 23:27:57,286 - __main__ - INFO - Starting KNN Evaluation
2024-08-29 23:27:57,286 - __main__ - ERROR - Error evaluating KNN model:name 'paramet_tune' is not defined
2024-08-29 23:27:57,286 - __main__ - INFO - Starting Random Forest Classifier Evaluation
2024-08-29 23:27:57,286 - __main__ - ERROR - Error evaluating Random Forest Classifier model:RandomForest_tuner.__init__() takes 3 positional arguments but 5 were given
2024-08-29 23:27:57,286 - __main__ - INFO - Starting XGBoost Classifier Evaluation
2024-08-29 23:27:57,286 - __main__ - ERROR - Error evaluating XGBoost Classifier model:name 'paramet_tune' is not defined
2024-08-29 23:27:57,287 - __main__ - INFO - Starting SVM Classifier Evaluation
2024-08-29 23:27:57,287 - __main__ - ERROR - Error evaluating SVM model:The 'scoring' parameter of cross_validate must be a str among {'neg_mean_poisson_deviance', 'mutual_info_score', 'balanced_accuracy', 'precision_macro', 'top_k_accuracy', 'f1', 'roc_auc_ovo', 'jaccard_macro', 'max_error', 'matthews_corrcoef', 'neg_mean_absolute_error', 'jaccard', 'fowlkes_mallows_score', 'normalized_mutual_info_score', 'r2', 'roc_auc_ovr_weighted', 'roc_auc', 'neg_mean_gamma_deviance', 'adjusted_rand_score', 'neg_mean_absolute_percentage_error', 'homogeneity_score', 'neg_root_mean_squared_log_error', 'neg_mean_squared_error', 'roc_auc_ovo_weighted', 'f1_samples', 'f1_macro', 'recall_micro', 'explained_variance', 'recall_samples', 'precision_samples', 'f1_micro', 'precision_micro', 'recall_weighted', 'neg_brier_score', 'roc_auc_ovr', 'precision_weighted', 'recall', 'neg_log_loss', 'f1_weighted', 'precision', 'neg_negative_likelihood_ratio', 'jaccard_samples', 'jaccard_weighted', 'rand_score', 'v_measure_score', 'neg_median_absolute_error', 'average_precision', 'jaccard_micro', 'adjusted_mutual_info_score', 'neg_mean_squared_log_error', 'completeness_score', 'recall_macro', 'positive_likelihood_ratio', 'accuracy', 'neg_root_mean_squared_error'}, a callable, an instance of 'list', an instance of 'tuple', an instance of 'dict' or None. Got array([1, 1, 2, ..., 1, 0, 2]) instead.
2024-08-29 23:27:57,287 - __main__ - INFO - Starting Naive Bayes Classifier Evaluation
2024-08-29 23:27:57,287 - __main__ - ERROR - Error evaluating Naive Bayes model:The 'scoring' parameter of cross_validate must be a str among {'neg_mean_poisson_deviance', 'mutual_info_score', 'balanced_accuracy', 'precision_macro', 'top_k_accuracy', 'f1', 'roc_auc_ovo', 'jaccard_macro', 'max_error', 'matthews_corrcoef', 'neg_mean_absolute_error', 'jaccard', 'fowlkes_mallows_score', 'normalized_mutual_info_score', 'r2', 'roc_auc_ovr_weighted', 'roc_auc', 'neg_mean_gamma_deviance', 'adjusted_rand_score', 'neg_mean_absolute_percentage_error', 'homogeneity_score', 'neg_root_mean_squared_log_error', 'neg_mean_squared_error', 'roc_auc_ovo_weighted', 'f1_samples', 'f1_macro', 'recall_micro', 'explained_variance', 'recall_samples', 'precision_samples', 'f1_micro', 'precision_micro', 'recall_weighted', 'neg_brier_score', 'roc_auc_ovr', 'precision_weighted', 'recall', 'neg_log_loss', 'f1_weighted', 'precision', 'neg_negative_likelihood_ratio', 'jaccard_samples', 'jaccard_weighted', 'rand_score', 'v_measure_score', 'neg_median_absolute_error', 'average_precision', 'jaccard_micro', 'adjusted_mutual_info_score', 'neg_mean_squared_log_error', 'completeness_score', 'recall_macro', 'positive_likelihood_ratio', 'accuracy', 'neg_root_mean_squared_error'}, a callable, an instance of 'list', an instance of 'tuple', an instance of 'dict' or None. Got array([1, 1, 2, ..., 1, 0, 2]) instead.
2024-08-29 23:27:57,287 - __main__ - INFO - Starting CatBoost Classifier Evaluation
2024-08-29 23:27:57,287 - __main__ - ERROR - Error evaluating CatBoost model:Catboost_tuner.__init__() takes 3 positional arguments but 5 were given
2024-08-29 23:27:57,287 - __main__ - INFO - Starting LOF Evaluation
2024-08-29 23:27:57,318 - __main__ - ERROR - Error evaluating LOF model:"None of [Index([    0,     1,     2,     3,     4,     5,     6,     7,     8,     9,\n       ...\n       78588, 78589, 78591, 78592, 78593, 78594, 78595, 78597, 78598, 78599],\n      dtype='int64', length=62880)] are in the [columns]"
2024-08-29 23:27:57,318 - __main__ - INFO - Starting PCA Evaluation
2024-08-29 23:27:57,373 - __main__ - ERROR - Error evaluating PCA model:multi_class must be in ('ovo', 'ovr')
2024-08-29 23:27:57,373 - __main__ - INFO - Starting Isolation Forest Evaluation
2024-08-29 23:27:57,373 - __main__ - ERROR - Error evaluating Isolation Forest model:name 'paramet_tune' is not defined
2024-08-29 23:27:57,373 - __main__ - INFO - Starting DBSCAN Evaluation
2024-08-29 23:27:57,384 - __main__ - ERROR - Error evaluating DBSCAN model:"None of [Index([    0,     1,     2,     3,     4,     5,     6,     7,     8,     9,\n       ...\n       78588, 78589, 78591, 78592, 78593, 78594, 78595, 78597, 78598, 78599],\n      dtype='int64', length=62880)] are in the [columns]"
2024-08-29 23:27:57,384 - __main__ - INFO - Starting COPOD Evaluation
2024-08-29 23:27:58,327 - __main__ - ERROR - Error evaluating COPOD model:multi_class must be in ('ovo', 'ovr')
2024-08-29 23:27:58,328 - __main__ - INFO - Starting ECOD Evaluation
2024-08-29 23:27:58,879 - __main__ - ERROR - Error evaluating ECOD model:multi_class must be in ('ovo', 'ovr')
2024-08-29 23:27:58,882 - __main__ - INFO - The evaluation results are saved to CSV file.
2024-08-29 23:51:10,013 - __main__ - INFO - Starting KNN Evaluation
2024-08-29 23:52:19,871 - __main__ - INFO - KNN Evaluation: ROC AUC SCORE=0.542, F1 SCORE=0.15, Runtime=69.856
2024-08-29 23:52:19,872 - __main__ - INFO - Starting Random Forest Classifier Evaluation
